uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/ProtocolHandlerServices/BlackBoxTest/ProtocolHandlerBBTestFunction_2.c

```diff
@@ -8145,8 +8147,8 @@ BBTestRegisterProtocolNotifyInterfaceTestCheckPoint6 (
   EFI_STATUS            Status1, Status2;
   UINTN                 EventIndex;
   NOTIFY_CONTEXT_2      NotifyContextArray[2];
-  NOTIFY_CONTEXT_2      FirstNotifyContextArray[2];
-  NOTIFY_CONTEXT_2      SecondNotifyContextArray[2];  
+  NOTIFY_CONTEXT_2      FirstNotifyContextArray[2] = {{EFI_SUCCESS, 0}};
+  NOTIFY_CONTEXT_2      SecondNotifyContextArray[2] = {{EFI_SUCCESS, 0}};

```

Comment:
Second element not initialized. 
Lets initialize all variable while definition.